src/finn/transformation/fpgadataflow/vitis_build.py

```diff
@@ -267,12 +276,13 @@ def apply(self, model):
             f.write(
                 "v++ -t hw --platform %s --link %s"
                 " --kernel_frequency %d --config config.txt --optimize %s"
-                " --save-temps -R2\n"
+                " --save-temps -R2 %s\n"
                 % (
                     self.platform,
                     " ".join(object_files),
                     self.f_mhz,
                     self.strategy.value,
+                    debug_commands,
```

Comment:
shouldn't this be something like `" ".join(debug_commands)`? otherwise we get a command line call that uses Python list syntax.

Reply:
Yes that's correct, I'll fix